rendered/manifests/agent-only/configmap-agent.yaml

```diff
@@ -172,7 +172,17 @@ data:
             - targets:
               - ${K8S_POD_IP}:8889
       receiver_creator:
-        receivers: null
+        receivers:
+          smartagent/kubernetes-apiserver:
+            config:
+              discoveryRule: target == "port" && port == 443 && pod.labels["k8s-app"] == "kube-apiserver"
```

Comment:
same here, remove this line

Reply:
oh the rest are rendered templates, nvm once you remove and make render this will be gone.